scripts/watermarkstat

```diff
@@ -56,7 +56,7 @@ class Watermarkstat(object):
         def get_queue_type(table_id):
             queue_type = self.counters_db.get(self.counters_db.COUNTERS_DB, COUNTERS_QUEUE_TYPE_MAP, table_id)
             if queue_type is None:
-                print >> sys.stderr, "Queue Type is not available!", table_id
+                sys.stderr.write("Queue Type is not available in table '{}'".format(table_id))
```

Comment:
How about `print(*args, file=sys.stderr)`

Reply:
I considered it, but this method results in a shorter line.

Reply:
Changed all instsances.